services/storage/app/controllers/files_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class FilesController < Storage::ApplicationController
+
+  def index
+    render status: 200, json: json_resources(FileResource, files)
+  end
+
+  def create
+    file = current_storage.upload! io: params[:file]
+    render status: 200, json: json_resource(FileResource, file)
+  end
+
+  private
+
+  def files
```

Comment:
doesn't seem like this method is used anywhere. is it needed?

Reply:
used for index action